tests/tool_result_rails_actions.py

```
@@ -30,14 +30,14 @@
 
 
 @action(is_system_action=True)
-async def self_check_tool_input(
+async def self_check_tool_result(
     tool_message: str = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)